### PR TITLE
Review

### PR DESCRIPTION
**Pull request by maiana-sanjuan**

### CODE REVIEW

**Review by rmccreath — COMMENTED** [review]
Great work! I've made some suggestions that are more around project structure but will help keep the project tidy and help with maintenance/reproducibility. It'll be good to separate the data collection/wrangling from the app stuff - I'll outline that below though.

I'm happy to help contribute to these changes. 

**Project Structure**
It can be difficult to foresee how a project will develop and how things should be set up. You've both done loads of work on this now though, so it's a bit clearer to see how that could look. 

Potential structure:
- Code - *all R scripts not directly related to the app, i.e. data collection/wrangling, should be collected together in this folder. Consider some naming convention and numbering if required, e.g. smr01_01_data_extract, smr01_02_data_wrangle*
    - Setup file - *there are a few places where the same packages are loaded, and similar processes followed. A global `setup_environment` file is common, this can then be loaded into any further scripts*
- App - *this will have the `global`, `ui`, and `server` files*. I'm not sure if everything that's happening in global should be moved to the code folder as part of that process though?
- Lookups - *as is. However, avoiding any performance issues it could be possible to read this directly from Open Data rather than storing additional files with the app*
- Data - *this will have all base data tables that are used by the app, and where all the outputs from `code` will go*
- Doc - *any documentation related to the project, minimal if possible*

**File Paths**
When using R projects, it shouldn't be necessary to use `setwd()`, it's also dangerous to have literal file paths (e.g. `~/dqa_dashboard/data/...`) as this doesn't work across systems. I've tried to catch these throughout.

**Code Style**
While this is less of an issue, code style moves between base R and using Tidyverse. Code that uses Tidyverse is often easier to maintain and is more efficient so it's best to stick to that where possible.

File: dashboard/SMR_accuracy.csv [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+"117","Other Conditions (3-digits)","2015/16",67,"SMR04"
+"118","Other Conditions (4-digits)","2015/16",62,"SMR04"
+"119","Status on Admission","2015/16",92,"SMR04"
+"120","Admission Date","2015/16",96,"SMR04"
+"121","Admission Type","2015/16",70,"SMR04"
+"122","Admission Reason","2015/16",75,"SMR04"
+"123","Admission Referral From","2015/16",85,"SMR04"
+"124","Discharge Date","2015/16",92,"SMR04"
+"125","Discharge Type","2015/16",91,"SMR04"
+"126","Discharge Transfer To","2015/16",84,"SMR04"
```
[the remark this answers is not in the document]
Comment (maiana-sanjuan) [reply dashboard/SMR_accuracy.csv:127]: agreed, will move it to the data folder :)

File: dqa_dasboard.Rproj [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+SaveWorkspace: Default
+AlwaysSaveHistory: Default
+
+EnableCodeIndexing: Yes
+UseSpacesForTab: Yes
+NumSpacesForTab: 2
+Encoding: UTF-8
+
+RnwWeave: Sweave
+LaTeX: pdfLaTeX
```
[the remark this answers is not in the document]
Comment (maiana-sanjuan) [reply dqa_dasboard.Rproj:13]: oops, yes I need to delete it

File: smr_audit.R [hunk 3]
```diff
@@ -0,0 +1,15 @@
+library(readxl)
+library(tidyverse)
+
```
[the remark this answers is not in the document]
Comment (maiana-sanjuan) [reply smr_audit.R:3]: apologies, was using some of this code to collate smr data in the earlier stages of the project but its no longer necessary ^^'

File: smr_data/healthboard.R [hunk 4]
```diff
@@ -0,0 +1,85 @@
+library(tidyr)
+library(dplyr)
+library(readxl)
+library(stringr)
+library(ggplot2)
+library(ggrepel)
+
```
[the remark this answers is not in the document]
Comment (maiana-sanjuan) [reply smr_data/healthboard.R:7]: apologies, I was using this script in the early stages of the dashboard project for planning but it is no longer necessary for the app. will do some cleaning up!

File: smr_audit.R [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+#path to SMR accuracy audit results
+hb_path <- "~/dqa_dashboard/smr_data/Hospital_SMR_accuracy_2004-Present.xlsx"
```
[the remark this answers is not in the document]
Comment (maiana-sanjuan) [reply smr_audit.R:5]: it works! thank you :)

File: smr_data/healthboard.R [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  coord_flip()+
+  theme_bw()+
+  labs(
+    title = "SMR01 Average Accuracy Percentage per Health Board",
+    y = "Percentage Accuracy (%)",
+    x = "Health Board"
+  )
+
+
+p+geom_hline(yintercept=88.6, color = "blue")
```
[the remark this answers is not in the document]
Comment (maiana-sanjuan) [reply smr_data/healthboard.R:81]: Apologies! I was using this for some visual ideas during the early stages of the project, it is no longer necessary to the running of the app

**Review by rmccreath — APPROVED** [review]

**Pull request merged**